File: Questionnaire.Bll/Dtos/ResultDtos/QuestionnaireResultListDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Questionnaire.Bll.Dtos.ResultDtos
{
    public class QuestionnaireResultListDto
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public DateTime Begining { get; set; }

        public DateTime Finish { get; set; }

        public int Questions { get; set; }

        public int Solved { get; set; }

        public int Members { get; set; }

        public ICollection<QuestionnaireResultHeaderDto> Results { get; set; }
    }
}

File: Questionnaire.Api/ClientApp/src/app/features/questionnaire/models/result/questionnaireResultListDto.ts
export interface QuestionnaireResultListDto {
    id: number;
    title: string;
    description: string;
    begining: Date;
    finish: Date;
    questions: number;
    solved: number;
    members: number;
    results: QuestionnaireResultHeaderDto[];
}

export interface QuestionnaireResultHeaderDto {
    id: number;
    userName: string;
    maximumPoints: number;
    points: number;
    start: Date;
    completedTime: Date;
    completed: boolean;
}
